app/views/hyrax/dashboard/sidebar/_activity.html.erb

```diff
@@ -4,12 +4,15 @@
     <%= menu.collapsable_section t('hyrax.admin.sidebar.repository_activity'),
                                  icon_class: "fa fa-line-chart",
                                  id: 'collapseRepositoryActivity',
-                                 open: menu.repository_activity_section? do %>
-      <%= menu.nav_link(hyrax.dashboard_path) do %>
+                                 open: menu.repository_activity_section?,
+                                 title: t('hyrax.admin.sidebar.repository_activity') do %>
```

Comment:
If we aren't adding alt tags now, could you make sure there's a separate ticket for it? I'm pretty sure, given the other alt tag failure, that not having alt tags on the menu will also be an issue.

Reply:
Ahh that's right. Ok!